booster/unit-tests/Test/Booster/Pattern/MatchRewrite.hs

```diff
@@ -376,6 +377,17 @@ internalLists =
 
     klist = KList testKListDef
 
+internalSets :: TestTree
+internalSets =
+    testGroup
+        "Internal sets"
+        [ test
+            "Can match an empty set with itself"
+            emptySet
+            emptySet
```

Comment:
Need to export `emptySet` from `Test.Booster.Pattern.InternalCollections`
Rest LGTM, we just have to fix this test.